suites/fs/basic/tasks/asok_dump_tree.yaml

```diff
@@ -0,0 +1,7 @@
+tasks:
+- install:
```

Comment:
You don't need the install and ceph bits here (they're in the inline/) tasks, perversely...